src/puppet-web/puppet-web.ts

```diff
@@ -353,6 +353,10 @@ export class PuppetWeb extends Puppet {
       }))
     })
 
+    let fileMaxSize = 20 * 1024 * 1024
```

Comment:
1. Should be const instead of let.
2. Could you link to the size limit code line in wxwebapp(we have a track tool named webwxapp-tracker
)? So we could track it in the future.